app/js/services/filter.service.js

```diff
@@ -391,16 +391,16 @@ function FilterService(
             filterService.setSelectedProduct(params.product);
         }
 
-        if (params.actions) {
-            if (params.actions === 'all') {
-                MultiButtonService.setState('inventoryWithoutActions', true);
-                MultiButtonService.setState('inventoryWithActions', true);
-            } else if (params.actions === 'with') {
+        if (params.systemHealth) {
+            if (params.systemHealth === 'affected') {
                 MultiButtonService.setState('inventoryWithoutActions', false);
                 MultiButtonService.setState('inventoryWithActions', true);
-            } else if (params.actions === 'without') {
+            } else if (params.systemHea === 'healthy') {
                 MultiButtonService.setState('inventoryWithoutActions', true);
                 MultiButtonService.setState('inventoryWithActions', false);
+            } else {
+                MultiButtonService.setState('inventoryWithoutActions', true);
+                MultiButtonService.setState('inventoryWithActions', true);
             }
         }
```

Comment:
I don't think the MultiButtonService is doing anything or systemHealth. When I remove it the system health filter works fine as far as I can tell.